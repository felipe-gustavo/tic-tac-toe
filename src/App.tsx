import { AppContainer, AppHeader } from "./App.style";
import { AppDescription } from "./components/molecules/AppDescription";
import { AppLogo } from "./components/molecules/AppLogo";

function App() {
  return (
    <AppContainer>
      <AppHeader>
        <AppLogo />
        <AppDescription />
      </AppHeader>
    </AppContainer>
  );
}

export default App;
