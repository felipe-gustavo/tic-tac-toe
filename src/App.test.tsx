import { fireEvent, render, screen } from "@testing-library/react";

import App from "./App";

describe("<App />", () => {
  it("match with the screenshot", () => {
    const { baseElement } = render(<App />);
    expect(baseElement).toMatchSnapshot();
  });
});
