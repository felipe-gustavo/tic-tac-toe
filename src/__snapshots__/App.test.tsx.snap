// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> match with the screenshot 1`] = `
<body>
  <div>
    <div
      class="sc-bczRLJ kIOTIl"
    >
      <header
        class="sc-gsnTZi xxDRN"
      >
        <div
          class="sc-hKMtZM kqqsny"
          rotate="0"
        >
          <svg
            class="logo"
            height="40vmin"
            role="img"
          >
            logo.svg
          </svg>
          <button
            class="sc-eCYdqJ iUfoxH primary switch-logo-button"
            type="button"
          >
            Spin logo
          </button>
        </div>
        <p>
          Edit 
          <code>
            src/App.tsx
          </code>
           and save to reload.
        </p>
        <a
          class="sc-dkzDqf gIyjLW"
          href="https://reactjs.org"
          rel="noopener noreferrer"
          target="_blank"
        >
          Learn React
        </a>
      </header>
    </div>
  </div>
</body>
`;
